lib/Event/AbstractManagerEvent.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Event;
+
+use Sulu\Component\DocumentManager\DocumentManagerInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Abstract class for events which require the DocumentManagerInterface - i.e.
+ * all events for which subscribers require access to a document manager or any
+ * of its dependencies.
+ */
+abstract class AbstractManagerEvent extends AbstractEvent
```

Comment:
Any real need for this class? Why not just combining it with the `AbstractDocumentEvent`? Or the `AbstractEvent`? This chain of inheritance gets quite long this way.